modules/virt-configuring-default-and-virt-default-storage-class.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assembly:
+//
+// * virt/storage/virt-automatic-bootsource-updates.adoc
+//
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-configuring-default-and-virt-default-storage-class_{context}"]
+= Configuring the default and virt-default storage classes
+
+A storage class determines how persistent storage is provisioned for workloads. In {VirtProductName}, the virt-default storage class takes precedence over the cluster default storage class and is used specifically for virtualization workloads. Only one storage class should be set as virt-default or cluster default at a time. If multiple storage classes are marked as default, the virt-default storage class overrides the cluster default. To ensure consistent behavior, configure only one storage class as the default for virtualization workloads.
+
+[IMPORTANT]
+====
+Boot sources are created using the default storage class. When the default storage class changes, old boot sources are automatically updated using the new default storage class. If your cluster does not have a default storage class, you must define one.
+
+If boot source images were stored as VolumeSnapshots and both the cluster default and `virt-default` storage class have been unset, the VolumeSnapshots will be cleaned up and new DataVolumes will be created. However they will not start importing until a default storage class is set.
+====
+
+.Procedure
+
+. Patch the current virt-default or a cluster-default storage class to false:
```

Comment:
Previously, [_cluster default_ didn't take a hyphen](https://github.com/openshift/openshift-docs/pull/88579/files#diff-f9e9945995f11e4d477ac976d85e6140e12c4b81e11e4e3369d778fa9382e06fR10), which seems right to me.